src/azul/plugins/hca.py

```diff
@@ -0,0 +1,53 @@
+from abc import (
```

Comment:
Can this live in `azul.plugins.metadata.hca`?

Reply:
Same as above